packages/app/src/components/SideBarNavigation/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Sidebar } from 'react-rainbow-components';
+import { useLocation } from 'react-router-dom';
+import StyledContainer from './styled';
+
+const SideBarNavigation = (props) => {
+    const { className, style, children } = props;
+    const location = useLocation();
+    const routes = React.Children.map(children, (element) => {
+        return {
+            path: element.props.path,
+            name: element.props.name,
+            exact: element.props.exact,
+        };
+    });
+    const currentRoute = routes.find((route) => {
+        if (route.exact) {
+            return route.path === location.pathname;
+        }
+        return location.pathname.startsWith(route.path);
+    });
```

Comment:
let's use useRouteMatch or match API here, there isn't a need to build our own logic for matching which is the selected item.

Reply:
useRouteMatch('some/path') will return null it the current location does not match path provided.